api/s2n.h

```diff
@@ -474,7 +474,7 @@ extern int s2n_get_cert_from_cert_chain(struct s2n_cert_chain_and_key *chain_and
  * @param cert_length This return value represents the length of the certificate.
  */
 S2N_API
-extern int s2n_get_cert_der(struct s2n_cert *cert, uint8_t **out_cert_der, uint32_t *cert_length);
+extern int s2n_get_cert_der(const struct s2n_cert *cert, uint8_t **out_cert_der, uint32_t *cert_length);
```

Comment:
I'm pretty sure there should be some `const` on the `out_cert_der` parameter to indicate to the caller that they are not permitted to modify its contents.